android/src/main/kotlin/com/qonversion/flutter/sdk/qonversion_flutter_sdk/QonversionFlutterSdkPlugin.kt

```diff
@@ -138,7 +141,9 @@ class QonversionFlutterSdkPlugin : MethodCallHandler, FlutterPlugin, ActivityAwa
                     }
             )
             startListeningPendingPurchasesEvents()
-        } ?: result.error(QonversionErrorCode.UnknownError.name, "Couldn't launch Qonversion. There is no Application context", null)
+            automationsPlugin?.setAutomationsDelegate()
+        }
+                ?: result.error(QonversionErrorCode.UnknownError.name, "Couldn't launch Qonversion. There is no Application context", null)
```

Comment:
Is this correct indent?

Reply:
It is a way like Android Studio takes care of the formatting code.